packages/pagination/src/usePagination.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useSelection } from '@zendeskgarden/container-selection';
+
+/**
+ * A React Hook used to create accessible widgets with the Pagination Interaction Pattern
+ *
+ * https://www.w3.org/TR/wai-aria-practices/#Listbox
+ */
+export function usePagination(options) {
+  const {
+    selectedItem,
+    focusedItem,
+    getContainerProps: getControlledContainerProps,
+    getItemProps
+  } = useSelection(options);
+
+  const getContainerProps = ({ ariaLabel, ...props } = {}) => {
```

Comment:
This should be getting applied to an outer `<nav>` throughout (not `<ul>`). As such, it would be good to give the function a `role = 'navigation'` that can be knocked out via `null` if the user knows they are using the correct semantic container element. See `useBreadcrumb` for a similar pattern.